Classes/Domain/Model/DocumentFormField.php

```diff
@@ -35,6 +35,8 @@ class DocumentFormField extends AbstractFormElement
 
     protected $dataType;
 
+    protected $linkedFieldUid;
```

Comment:
Variable data type is missing.

Reply:
The type annotation above the declaration is used by extbase for validation of this object.
E.g.  
<pre>
 /**
  * @var string
  */
</pre>

The same for getter and setter.